src/Types.jl

```diff
@@ -1390,11 +1389,11 @@ function find_registered!(env::EnvCache,
 end
 
 find_registered!(env::EnvCache, uuids::Vector{UUID}; force::Bool=false)::Nothing =
-    find_registered!(env, String[], uuids, force=force)
+    find_registered!(env, String[], uuids)
 
-# Lookup all packages in project & manifest files
-find_registered!(env::EnvCache)::Nothing =
-    find_registered!(env, String[], UUID[], force=true)
+# # Lookup all packages in project & manifest files
+# find_registered!(env::EnvCache)::Nothing =
+#     find_registered!(env, String[], UUID[])
```

Comment:
Just delete, don't comment out.

Reply:
Yea, was just trying it out, but seems CI is happy so will clean up.